<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperateCurveClass</class>
 <widget class="QMainWindow" name="OperateCurveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>763</width>
    <height>589</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>765</width>
    <height>595</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OperateCurve</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>371</width>
            <height>91</height>
           </size>
          </property>
          <property name="title">
           <string>Curve Control</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Curve Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboCurveType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="checkForceCorrectness">
                 <property name="text">
                  <string>Force Correctness on Drag</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_18">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>361</width>
            <height>91</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>361</width>
            <height>91</height>
           </size>
          </property>
          <property name="title">
           <string>Adjust</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="QLabel" name="label_48">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="editX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_49">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLineEdit" name="editY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Value Range:[0, 4096]</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="btnSetValue">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Set</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="labelPosition">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Position: Out of chart</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CurveChartView" name="widgetChart" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>741</width>
          <height>471</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurveChartView</class>
   <extends>QWidget</extends>
   <header location="global">curvechartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="OperateCurve.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboCurveType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OperateCurveClass</receiver>
   <slot>curve_type_combo_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkForceCorrectness</sender>
   <signal>clicked(bool)</signal>
   <receiver>OperateCurveClass</receiver>
   <slot>force_correctness_on_drag_checked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetValue</sender>
   <signal>clicked()</signal>
   <receiver>OperateCurveClass</receiver>
   <slot>set_value_on_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>curve_type_combo_changed(int)</slot>
  <slot>force_correctness_on_drag_checked(bool)</slot>
  <slot>set_value_on_clicked()</slot>
 </slots>
</ui>
